app/controllers/pets_controller.rb

```diff
@@ -53,13 +53,20 @@ def update_pending_status
 
   def destroy
     Pet.destroy(params[:id])
+    if favorite.contents.include? params[:id]
+      favorite.remove_pet(params[:id])
+    end
```

Comment:
Ya, I think this can be re-written as `params[:id].each {|id| favorite.contents.delete(id)}` since the `delete` method can take an array of id's as an argument. This also probably means that  our `remove_pet` method is unnecessary. It's worth trying out before merging, but if it doesn't immediately work, I'm fine with leaving this as is.

Reply:
Since we utilize favorite.add_pet, for me it makes sense to have a remove_pet as well. Even though it calls contents.delete, remove_pet on favorite is more explicit imo.